lib/tasks/stickers.rake

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+namespace :gift do
+  desc 'gift remaining stickers to incompleted users'
+  task stickers: :environment do
```

Comment:
I would make this rake task generic, not focused on stickers just on the process of gifting.